src/flow-manager.c

```diff
@@ -975,17 +975,20 @@ static TmEcode FlowManager(ThreadVars *th_v, void *thread_data)
             timeradd(&cond_tv, &add_tv, &cond_tv);
 
             struct timespec cond_time = FROM_TIMEVAL(cond_tv);
-            SCCtrlMutexLock(&flow_manager_ctrl_mutex);
             while (1) {
+                SCCtrlMutexLock(&flow_manager_ctrl_mutex);
```

Comment:
I think this is wrong. The coverity issue appears to be that there are checks after the timedwait, which should move before the timedwait